great_expectations/expectations/metrics/map_metric_provider/column_pair_condition_partial.py

```diff
@@ -40,7 +39,7 @@
 
 def column_pair_condition_partial(  # noqa: C901 - 16
     engine: Type[ExecutionEngine],
-    partial_fn_type: Optional[Union[str, MetricPartialFunctionTypes]] = None,
+    partial_fn_type: Optional[MetricPartialFunctionTypes] = None,
```

Comment:
Was this a `str` before because users could define custom metric types that wouldn't be part of the enum?